repo.conf

```diff
@@ -0,0 +1,7 @@
+### Configuration for syncing git repositories ###
```

Comment:
Instead of including this file, maybe include a `repo.conf.example` file - that way if you want to make updates to the way this file works, an end-user will be able to see those without conflicting with their local changes